samples/sample-app/package.json

```diff
@@ -1,44 +1,45 @@
 {
-  "name": "sample-app",
-  "version": "0.1.0",
+  "name": "test-sampleapp",
+  "version": "1.1.0",
   "private": true,
   "scripts": {
     "serve": "vue-cli-service serve",
     "build": "vue-cli-service build",
     "lint": "vue-cli-service lint"
   },
   "dependencies": {
-    "imagekitio-vue": "^1.0.7",
-    "vue": "^2.6.10"
+    "core-js": "^3.8.3",
+    "imagekit-javascript": "^1.3.5",
+    "imagekitio-vue": "file:../../",
```

Comment:
Please include the future (to be published) version number and remove package-lock.json file. Also why so we need sample-app here, we have dedicated quick start guides for all SDKs - https://github.com/imagekit-samples/quickstart/tree/master/vuejs. Is it just for test suite?